bpmn-intellij-plugin-core/src/main/kotlin/com/valb3r/bpmn/intellij/plugin/core/advertisement/AdvertisementState.kt

```diff
@@ -0,0 +1,30 @@
+package com.valb3r.bpmn.intellij.plugin.core.advertisement
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.openapi.project.Project
+import java.time.LocalDate
+
+@State(name = "AdvertisementState", storages = [(Storage("opensource-polybpmn-advertisement.xml"))])
+class AdvertisementState: PersistentStateComponent<AdvertisementState.State> {
```

Comment:
This state is per-project. We need global state like BaseBpmnPluginSettingsState